r/rsparkling/tests/testthat.R

```diff
@@ -4,5 +4,11 @@ library(sparklyr)
 library(h2o)
 library(rsparkling)
 
-test_check("rsparkling")
+options(rsparkling.sparklingwater.version = Sys.getenv("SPARKLINGWATER_VERSION","1.6.7"),
+        rsparkling.spark.version =          Sys.getenv("SPARK_VERSION","1.6.2"))
 
+if(identical(Sys.getenv("NOT_CRAN"), "true")) { # testthat::skip_on_cran
```

Comment:
`testthat::skip_on_cran` must be used locally in each tests, I extract exact logic of it and put it here so it works globally on all tests now.